script.js

```diff
@@ -1,4 +1,122 @@
+
+// add a win loss record, output draws and winning percentage
+
+var STONE = `stone`;
+var SCISSORS = `scissors`;
+var PAPER = `paper`;
```

Comment:
This is good practice. Firstly, you've assigned the string constants to variables, and secondly, you're using screaming snake case, which has the effect of visually separating variables for string constants, from the other variables.